source/administration/command-line-tools.rst

```diff
@@ -358,6 +359,29 @@ mattermost channel restore
 
       ./mattermost channel restore 8soyabwthjnf9qibfztje5a36h
       ./mattermost channel restore myteam:mychannel
+      
+mattermost channel search
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+  Description
+    Search for a channel by channel name. 
+    
+  Format
+    .. code-block:: none
+
+      mattermost channel search {channelName}
+
+  Examples
+    .. code-block:: none
+
+      ./mattermost channel search mychannel
```

Comment:
Are the accepted values for `mychannel` channel name only, or also channel id?

Reply:
Channel id is not supported at this time.